bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -240,14 +240,11 @@ spec:
             postStart:
               exec:
                 command:
-                  - /bin/bash
-                  - -ec
-                  - |
-                    until rabbitmqctl cluster_status >/dev/null; do
-                        echo "Waiting for cluster readiness..."
-                        sleep 5
-                    done
-                    rabbitmq-queues rebalance "all"
+                  - /opt/bitnami/scripts/rabbitmq/nodeshutdown.sh
+                  - -t
+                  - {{ .Values.terminationGracePeriodSeconds | quote }}
+                  - -d
+                  - {{ ternary "true" "false" .Values.image.debug | quote }}
```

Comment:
You mentioned it should be a preStop hook but I see it in the postStart hook. Could you double check this?

Reply:
True (I can't use this chart anymore since I can't use helm 3 yet. Unfortunately I made a copy-paste error from the chart I actually use.)